commands/active.go

```diff
@@ -21,3 +27,38 @@ func cmdActive(c *cli.Context) {
 		fmt.Println(host.Name)
 	}
 }
+
+func getActiveHost(store persist.Store) (*host.Host, error) {
+	hosts, err := listHosts(store)
+	if err != nil {
+		return nil, err
+	}
+
+	hostListItems := getHostListItems(hosts)
+
+	for _, item := range hostListItems {
+		if item.Active {
+			h, err := loadHost(store, item.Name)
+			if err != nil {
+				return nil, err
+			}
+			return h, nil
+		}
+	}
+
+	return nil, errors.New("Active host not found")
+}
+
+// IsActive provides a single function for determining if a host is active
+// based on both the url and if the host is stopped.
+func isActive(h *host.Host, currentState state.State, url string) (bool, error) {
```

Comment:
I moved this from `ls.go` into `active.go` ... because it seemed like it belonged here? But it's not directly referenced from here, so I might revert this. Part of the reason is I wanted the unit tests to go in `active_test.go` so it seemed right to have the function under test in the associated `active.go`...

So many interdependencies in the `commands` package!